prime-dotnet-webapi/Controllers/GisController.cs

```diff
@@ -175,6 +175,10 @@ public async Task<ActionResult> LdapLogin(int gisId, LdapLoginPayload payload)
             {
                 return Ok();
             }
+            if (ldapResponse.Authenticated == "true")
+            {
+                return Forbid();
```

Comment:
perhaps a quick comment here saying that the user is authenticated but does not have the role?

Reply:
Thanks @james-hollinger updated